### PR TITLE
Use task killing mode in TasksController

### PR DESCRIPTION
**Pull request by alenkacz**
Summary:
At some point, TaskKillingMode directive was introduced. Now it is used even in TasksController.

JIRA issues:

### CODE REVIEW

File: src/main/scala/mesosphere/marathon/api/akkahttp/v2/TasksController.scala [hunk 1]
```diff
@@ -92,32 +92,29 @@ class TasksController(
...
-
-                if (scale) {
-                  onSuccess(killAndScale(tasksByAppId, force)) { deploymentResult =>
-                    complete((StatusCodes.OK, List(Headers.`Marathon-Deployment-Id`(deploymentResult.deploymentId)), deploymentResult))
-                  }
-                } else {
-                  onSuccess(kill(tasksByAppId, instanceIdsToAppId, wipe)) { tasks =>
-                    complete(TasksList(tasks).toRaml)
-                  }
+                onSuccess(kill(tasksByAppId, instanceIdsToAppId, taskKillingMode == TaskKillingMode.Wipe)) { tasks =>
```
Comment (meln1k) [inline src/main/scala/mesosphere/marathon/api/akkahttp/v2/TasksController.scala]: What if we want to just kill the task?

Comment (meln1k) [reply src/main/scala/mesosphere/marathon/api/akkahttp/v2/TasksController.scala]: And it seems we don't have a test for this case (kill without wipe)

Comment (alenkacz) [reply src/main/scala/mesosphere/marathon/api/akkahttp/v2/TasksController.scala]: well that is not a problem, right? TaskKillingMode will not be Wipe, false will be passed to the kill method and the task will be killed without wipe...

Comment (meln1k) [reply src/main/scala/mesosphere/marathon/api/akkahttp/v2/TasksController.scala]: well, yep, but I'd write it a little bit more explicit :)

**Pull request merged**